### PR TITLE
Issue Notice: Undefined index in /wp-content/plugins/give/templates/shortcode-receipt.php on line 38

### PR DESCRIPTION
**Pull request by dixitadusara**
@DevinWalker 

I have tried to solve the undefined error for #1763. please review it and let me know if any changes.

Thanks,

### CODE REVIEW

**Comment by ravinderk** [comment]
@dixitadusara Thanks for creating a PR.

I reviewed this PR. Basically, if completed payment deleted then it should show notice. but currently, it was showing broken receipt.
screen shot 2017-06-08 at 8 53 56 pm

But I was expecting a notice if you notice in code:
screen shot 2017-06-08 at 8 54 23 pm


And also use this branch for resolving and creating PR:
https://github.com/WordImpress/Give/tree/release/1.8.9

File: templates/shortcode-receipt.php [hunk 1]
```diff
@@ -23,6 +23,8 @@
 $email        = give_get_payment_user_email( $payment->ID );
 $status       = $payment->post_status;
 $status_label = give_get_payment_status( $payment, true );
+$first_name = ! empty( $user ) && isset( $user['first_name'] ) ? $user['first_name'] : '';
```
Comment (ravinderk) [inline templates/shortcode-receipt.php]: @dixitadusara I am seeing code formatting issue here.
Please use WordPress coding standard for code formatting.
https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards

File: templates/shortcode-receipt.php [hunk 2]
```diff
@@ -17,6 +17,13 @@
 	return;
 }
 
+//Validation: Return, if receipt id is 0.
```
Comment (ravinderk) [inline templates/shortcode-receipt.php]: @dixitadusara There are no need to add extra code here check my above comment

**Comment by DevinWalker** [comment]
Thanks @dixitadusara - I'm closing in favor of: https://github.com/WordImpress/Give/pull/1766

**Pull request closed without merging**